src/main/webapp/style.css

```diff
@@ -0,0 +1,23 @@
+.row {
```

Comment:
Consider using css grids to create the columns instead so it can be responsive
[https://travishorn.com/responsive-grid-in-2-minutes-with-css-grid-layout-4842a41420fe](url)